internal/storetest/storedata.go

```diff
@@ -82,3 +91,74 @@ func (storeData *StoreData) LoadModel(basePath string) (authorizationmodel.Model
 
 	return format, nil
 }
+
+func (storeData *StoreData) LoadTuples(basePath string) error {
+	var errs error
+
+	if storeData.TupleFile != "" {
+		tuples, err := readTupleFile(path.Join(basePath, storeData.TupleFile))
+		if err != nil {
+			errs = fmt.Errorf("failed to process global tuple %s file due to %w", storeData.TupleFile, err)
+		} else {
+			storeData.Tuples = tuples
+		}
+	}
+
+	for index := 0; index < len(storeData.Tests); index++ {
+		test := storeData.Tests[index]
+		if test.TupleFile == "" {
+			continue
+		}
+
+		tuples, err := readTupleFile(path.Join(basePath, test.TupleFile))
+		if err != nil {
+			errs = errors.Join(
+				errs,
+				fmt.Errorf("failed to process tuple file %s for test %s due to %w", test.TupleFile, test.Name, err),
+			)
+		} else {
+			storeData.Tests[index].Tuples = tuples
+		}
+	}
+
+	if errs != nil {
+		return errors.Join(errors.New("failed to process one or more tuple files"), errs) //nolint:goerr113
```

Comment:
Given there was already precedence for suppressing `goerr113` and `wrapcheck` in the repo I suppressed the errors here. Should we just disable that linter?

Reply:
I was hoping to get back and fix these at some point, but yeah I don't know how much that makes sense.

I'm OK with disabling, but let's do that in a separate PR and maybe we can discuss how we should tackle these?